doc/api/n-api.md

```diff
@@ -3169,6 +3251,7 @@ support it:
 [Working with JavaScript Properties]: #n_api_working_with_javascript_properties
 [Working with JavaScript Values]: #n_api_working_with_javascript_values
 [Working with JavaScript Values - Abstract Operations]: #n_api_working_with_javascript_values_abstract_operations
+[Section 24.3]: https://tc39.github.io/ecma262/#sec-dataview-objects
```

Comment:
Can you move this up a few lines to keep the links sorted. This should come after the Section 12.5.5 link.